pysonnet/projects.py

```diff
@@ -405,7 +407,103 @@ def make_sonnet_file(self, file_path, clean=True):
         if not os.path.isdir(subfolder):
             os.mkdir(subfolder)
         log.debug("geometry project saved")
-
+   
+	def current_desnity_xml(**kwargs):
```

Comment:
- fix indentation
- function names in this repository follow the format `verb_noun()`. Change to `def export_current_density(self, kwargs):`
- explicitly define all keyword arguments in the function definition to stay consistent with the rest of the repository